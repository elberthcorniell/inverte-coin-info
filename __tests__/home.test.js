import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import store from '../src/reducers';

describe('test', () => {
  test('Render as spected', () => {
    render(<Provider store={store} />);
    expect(document.body.innerHTML).toMatchSnapshot();
  });
});
