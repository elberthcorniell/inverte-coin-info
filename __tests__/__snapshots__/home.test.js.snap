// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Render as spected 1`] = `"<div></div>"`;
